src/Arcus.Security.Tests.Unit/Core/SecretProviderCachingExtensionsTests.cs

```diff
@@ -1,11 +1,8 @@
-﻿using System;
```

Comment:
Probably some duplication of tests here since the obsolete methods are removed.

Reply:
done